src/engine/ruleset/environments/wazuh-environment.yml

```diff
@@ -15,48 +15,52 @@ decoders:
 
   # Internal decoders
   - decoder/dbsync/0
-  - decoder/fim/0
   - decoder/fim-event/0
   - decoder/fim-scan/0
+  - decoder/fim/0
+  - decoder/fortinet-client-endpoint/0
+  - decoder/fortinet-firewall/0
+  - decoder/fortinet-fortimail/0
+  - decoder/fortinet-fortimanager/0
   - decoder/redis/0
   - decoder/rootcheck/0
   - decoder/sca/0
   - decoder/syscollector-base/0
-  - decoder/syscollector-dbsync-network-protocol/0
-  - decoder/syscollector-dbsync-osinfo/0
-  - decoder/syscollector-dbsync-osinfo-inserted/0
-  - decoder/syscollector-dbsync-ports/0
-  - decoder/syscollector-dbsync-ports-inserted/0
-  - decoder/syscollector-dbsync-processes/0
-  - decoder/syscollector-dbsync-processes-inserted/0
```

Comment:
why changing the environment in decoders outside of the scope of this particular issue?
If it was in order to order them alphabetically, let's discuss the need with all the teams using the engine.
But the change should be done in another PR